File: programacao/c/respostas/selecao/40.c
#include <stdio.h>

int main(void) {
    float triglicerideos, hdl, razao;

    printf("Triglicerideos (mg/dL): ");
    scanf("%f", &triglicerideos);
    printf("HDL (mg/dL): ");
    scanf("%f", &hdl);

    razao = triglicerideos / hdl;

    printf("Razao: %.1f\n", razao);

    if (razao < 2.0) {
        printf("Baixo risco\n");
    } else if (razao <= 3.0) {
        printf("Risco moderado\n");
    } else {
        printf("Alto risco\n");
    }

    return 0;
}

File: programacao/python/respostas/selecao/40.py
triglicerideos = float(input("Triglicerideos (mg/dL): "))
hdl = float(input("HDL (mg/dL): "))

razao = triglicerideos / hdl

print(f"Razao: {razao:.1f}")

if razao < 2.0:
    print("Baixo risco")
elif razao <= 3.0:
    print("Risco moderado")
else:
    print("Alto risco")
